e2e/workspaces/wikipedia/subflows/onboarding-android.yaml

```diff
@@ -2,6 +2,9 @@ appId: org.wikipedia
 ---
 - launchApp:
     clearState: true
+- tapOn:
+      text: "Non existent view"
+      optional: true
```

Comment:
What's the value in this test compared to the one in android-advanced-flow.yaml?

Reply:
It's the same do you feel to add optional: false as well?

Reply:
I don't think that an explicit test for `optional: false` is necessary - it's well tested in the serialisation that false is the default option.

My point was more ... Why have the same test in both locations?

Reply:
Running tap from main flow and from composite command like runFlow are 2 different tests. We want to test for command present in both main flow and runFlow.